src/database/database.py

```
@@ -60,7 +93,113 @@ def _initialize_database(self):
             if conn:
                 conn.close()
 
-# Provide a way to access the singleton instance
+    def create_player(self, name, member_id, guild_id, channel_id, type_):
+        if isinstance(type_, PlayerType):
+            type_str = type_.value
+        else:
+            type_str = str(type_)
```

Ruff (0.8.2)

Use ternary operator `type_str = type_.value if isinstance(type_, PlayerType) else str(type_)` instead of `if`-`else`-block

Replace `if`-`else`-block with `type_str = type_.value if isinstance(type_, PlayerType) else str(type_)`

(SIM108)